resolwe_bio/tests/processes/test_alignment.py

```diff
@@ -295,7 +295,7 @@ def test_subread(self):
         inputs = {'genome': genome.id, 'reads': reads.id}
         aligned_reads = self.run_process('alignment-subread', inputs)
         self.assertFile(aligned_reads, 'stats', 'subread_reads_report.txt')
-        self.assertFile(aligned_reads, 'bam', 'subread_paired.bam')
+        self.assertFile(aligned_reads, 'bam', 'subread_single.bam')
```

Comment:
Does `subread_single.bam` exist in `Files` folder? Delete `subread_paired.bam` from `Files if necessary

Reply:
I have added both files in tests.